### ISSUE TITLE
Add support for XP-Pen Artist Pro 12

### ISSUE THREAD

**Issue by Izerpizer** [InfinityGhost/OpenTabletDriver#669]
# Tablet Information

| Name | Value |
| --------------- | ----- |
| Manufacturer   | XP-Pen |
| Product Name  | Artist Pro 12 |
| Width          |13.83"|
| Height          |8.87" 
| Pressure Levels | 8192|
| LPI             | I'm not sure what this is|

## Diagnostic Information

[export-diagnostics.txt](https://github.com/InfinityGhost/OpenTabletDriver/files/5840040/export-diagnostics.txt)

**Comment by Mrcubix** [comment]
For diag, in OpenTabletDriver.UX go to help > export diagnostic

**Comment by Izerpizer** [comment]
@Mrcubix Do you just want the device information for the tablet, or would you like me to post the full contents  of the file?

**Comment by Izerpizer** [comment]
I'm going to guess its the full file

**Comment by Mrcubix** [comment]
full file ofc

**Comment by Izerpizer** [comment]
@Mrcubix it has now been added.

**Comment by Izerpizer** [comment]
@Mrcubix out of curiosity, how long do you think it would take to add support for this XP-Pen tablet?

**Comment by X9VoiD** [comment]
@Izerpizer use the latest release (0.5.0) for getting the diagnostics and uploading the diagnostics as a file is preferred than copy pasting its contents.

**Comment by Izerpizer** [comment]
@X9VoiD do I have to manually update to 0.5? I did "sudo apt update" and the OpenTabletDriver Package was still listed as I think it was 0.4.2.

**Comment by X9VoiD** [comment]
https://github.com/InfinityGhost/OpenTabletDriver/releases/tag/v0.5.0

**Comment by Izerpizer** [comment]
@X9VoiD  so i installed OpenTabletDriver 0.5.0, and it instantly crashes when i try to open the gui

**Comment by Izerpizer** [comment]
@X9VoiD @Mrcubix , i have updated the diagnostic information, and uploaded it in file format.

side note, out of curiosity, in the installation guide, it doesnt say how to add open tablet driver as a repository for my package manager. So if i want to update the driver i seemingly have to manually update it.

**Comment by X9VoiD** [comment]
Extract [Artist Pro 12.zip](https://github.com/InfinityGhost/OpenTabletDriver/files/5855291/Artist.Pro.12.zip) into `Configurations\XP-Pen\Artist Pro 12.json` and see if it gets detected.

**Comment by Izerpizer** [comment]
@X9VoiD Alright I added the configuration as you described and it did initially get detected; however, I rebooted, and all of a sudden it was not able to detect the tablet any more. The `Artist Pro 12.json` file is still in the configuration folder after reboot, and the console shows that OpenTabletDriver tried searching for it, but it wasn't able to detect it. It's strange, it detected it fine at first, and then the driver crashed, so I rebooted, and now it doesn't detect it at all even though I have changed nothing.
